helm/cluster-api-provider-aws/values.yaml

```diff
@@ -3,7 +3,8 @@ name: cluster-api-provider-aws
 # needed. Please read https://github.com/giantswarm/cluster-api-provider-aws/blob/main/README.md on how to create a
 # release. Please include the short commit SHA in the tag name, such as `v2.0.2-gs-123abcd`. After changing this
 # tag, please run `make generate` to update CRDs and other manifests.
-tag: v2.3.0-gs-13a6101cc  # upstream v2.3.0 + backported features/fixes (https://github.com/giantswarm/cluster-api-provider-aws/pull/576) + more backports (https://github.com/giantswarm/cluster-api-provider-aws/pull/580 + https://github.com/giantswarm/cluster-api-provider-aws/pull/582 + https://github.com/giantswarm/cluster-api-provider-aws/pull/583)
+
```

Comment:
This should add a PR link

Reply:
Done, I also changed the format so it's a bit clearer